core/src/test/java/kafka/test/api/ShareConsumerTest.java

```diff
@@ -536,13 +536,12 @@ public void testExplicitAcknowledgementCommitAsync() throws InterruptedException
         shareConsumer1.acknowledge(secondRecord);
         shareConsumer1.commitAsync();
 
-        // Allowing acquisition lock timeout to expire.
-        Thread.sleep(20000);
```

Comment:
There are 2 other places where we do `Thread.sleep()` in the class waiting for the acquisition lock to expire, are those tests flaky as well. If so, we might need to change those as well.
1. https://github.com/apache/kafka/pull/17515/files#diff-e5460ef36eb7605c66769f33acde94ce23c662c6a7fecee0a54d74a062fc0210R1148
2. https://github.com/apache/kafka/pull/17515/files#diff-e5460ef36eb7605c66769f33acde94ce23c662c6a7fecee0a54d74a062fc0210R309